### PR TITLE
EN 976 groups add page

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

File: test/api/groups.test.js [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+  method: METHODS.GET,
   mockResponse: LIST_GROUPS_OK,
   useAuthentication: true,
 };
 
 jest.unmock('api/groups');
 jest.mock('api/apiManager', () => ({
-  makeRequest: jest.fn(() => new Promise(resolve => resolve({}))),
-  METHODS: { GET: 'GET' },
+  makeMockRequest: jest.fn(() => new Promise(resolve => resolve({}))),
```
Comment (alessioscalici) [inline test/api/groups.test.js]: Why using makeMockRequest directly?

File: test/ui/groups/add/AddFormContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import 'test/enzyme-init';
+import { mapDispatchToProps } from 'ui/groups/add/AddFormContainer';
+import { change } from 'redux-form';
+
+jest.mock('state/groups/actions', () => ({
+  sendPostGroup: jest.fn().mockReturnValue('sendPostGroup_result'),
+}));
+
+jest.mock('redux-form', () => ({
```
Comment (alessioscalici) [inline test/ui/groups/add/AddFormContainer.test.js]: Redux-form is already mocked globally, please remove local mock

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Pull request merged**